app/clients/sms/twilio.py

```diff
@@ -24,6 +24,15 @@ def get_twilio_responses(status):
 class TwilioSMSClient(SmsClient):
     def __init__(self, account_sid=None, auth_token=None, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        # Twilio docks at.
+        # https://www.twilio.com/docs/usage/webhooks/webhooks-connection-overrides
```

Comment:
Please change to  "Twilio docs" not docks.

Reply:
Muscle memory. Too much time typing about boats. D'oh!